nodes/inputs/ScSingleVertex.py

```diff
@@ -0,0 +1,34 @@
+import bpy
+
+from bpy.types import Node
+from .._base.node_base import ScNode
+from .._base.node_input import ScInputNode
+
+
+class ScSingleVertex(Node, ScInputNode):
+    bl_idname = "ScSingleVertex"
+    bl_label = "Single Vertex"
+
+    def init(self, context):
+        super().init(context)
+
+    def error_condition(self):
+        return (
+            super().error_condition()
+        )
+
+    def functionality(self):
+        name = self.inputs["Name"].default_value
+        me = bpy.data.meshes.new(name + "_mesh")
+        obj = bpy.data.objects.new(name, me)
+        obj.location = bpy.context.scene.cursor.location
```

Comment:
Leave the object at 0,0,0 instead of cursor location
Can be later changed using "Set Transform" if required

Reply:
> Leave the object at 0,0,0 instead of cursor location
> Can be later changed using "Set Transform" if required

I could check that when creating any object with the sorcar nodes, the behavior is that it is created where you have the cursor. To have a greater consistent with the other nodes, I thought it was a good idea to make it behave like the other nodes.

Reply:
Oh right, I forgot that primitives are added on the cursor...
I'll uncomment the code